WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -13,6 +13,12 @@ struct HubMenu: View {
     @State private var showingReviews = false
     @State private var showingCoupons = false
 
+    /// State to disable multiple taps on menu items
+    /// Make sure to reset the value to false after dismissing sub-flows
+    @State private var shouldDisableItemTaps = false
+
+    private let timer = Timer.publish(every: 1, on: .main, in: .common).autoconnect()
```

Comment:
What do you think about adding a comment about this timer?

Reply:
Sure - updated in f0cb2cb0536099928614f1fdd8943cc76a9234db.